### PR TITLE
[Doc] Update JSON.md

### PR DESCRIPTION
**Pull request by alberttwong**
fixed examples 2 and 5.
Signed-off-by: Albert Wong <atwong@alumni.uci.edu>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [x] 2.4
  - [x] 2.3

### CODE REVIEW

File: docs/sql-reference/sql-statements/data-types/JSON.md [hunk 1]
```diff
@@ -101,18 +101,15 @@ Example 2: Filter the data of the JSON column to retrieve the data that meets th
 
 ```Plain%20Text
 mysql> select * from tj where j->'a' = parse_json('1');
-+------+---------------------+
-|   id |         j           |
-+------+---------------------+
-|   1  | {"a": 1, "b": true} |
-+------+---------------------+
+Empty set (0.05 sec)
```
Comment (mofeiatwork) [inline docs/sql-reference/sql-statements/data-types/JSON.md:104]: this example is used to show the ability of json predicate, why remove it ?

Comment (alberttwong) [reply docs/sql-reference/sql-statements/data-types/JSON.md:104]: I'm not removing it.   I'm saying it's wrong and replacing it with the correct syntax.

**Review by murphyatwork — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by evelynzhaojie** [comment]
@mergify backport branch-3.0 branch-2.5 branch-2.4 branch-2.3

**Comment by evelynzhaojie** [comment]
@mergify backport branch-3.0

**Pull request merged**